torch/utils/_sympy/value_ranges.py

```diff
@@ -722,40 +836,46 @@ def sin(x):
 
     @staticmethod
     def sinh(x):
-        return ValueRanges.increasing_map(x, OpaqueUnaryFn_sinh)
+        # return ValueRanges.increasing_map(x, OpaqueUnaryFn_sinh)
+        return ValueRanges(-sympy.oo, sympy.oo)
```

Comment:
This value range rule chokes when you do sinh(sys.maxint). But this is pretty pointless. There's no need for accurate VR for non sizey float compute, so don't implement it.